Mage/src/main/java/mage/abilities/effects/common/combat/CantBeBlockedAttachedEffect.java

```diff
@@ -10,14 +10,20 @@
 /**
  * @author North
  */
-public class CantBeBlockedAttachedEffect extends RestrictionEffect {
+public class CantBeBlockedAttachedEffect extends EvasionEffect {
 
     public CantBeBlockedAttachedEffect(AttachmentType attachmentType) {
         super(Duration.WhileOnBattlefield);
-        this.staticText = attachmentType.verb() + " creature can't be blocked";
+
+        this.staticCantBeBlockedMessage = "can't be blocked";
+        this.staticText =
+            new StringBuilder(attachmentType.verb())
+                .append(" creature ")
+                .append(this.staticCantBeBlockedMessage)
+                .toString();
```

Comment:
Just for info: don't use `StringBuilder` in new code, it [useless](https://stackoverflow.com/q/1532461/1276632) for performance optimization in 99% and harder to read and support compared to simple concatenation (+).